qiskit/algorithms/evolvers/pvqd/pvqd.py

```diff
@@ -149,6 +149,7 @@ def __init__(
                 a random vector with elements in the interval :math:`[-0.01, 0.01]`.
             quantum_instance: The backend or quantum instance used to evaluate the circuits.
         """
+        super().__init__()
```

Comment:
I am not quite sure why the unreleased PVQD is being changed, but since this will be deleted/moved to time_evolvers to be released I guess it does not matter.